e2e/test-component/scenarios/embedding-sdk/tooltip-reproductions.cy.spec.tsx

```diff
@@ -59,53 +52,56 @@ describeEE("scenarios > embedding-sdk > tooltip-reproductions", () => {
     );
   });
 
-  // This is flaking on CI but not locally :()
-  it.skip("should render tooltips below the screen's height (metabase#51904)", () => {
-    cy.get("@dashboardId").then(dashboardId => {
-      mountSdkContent(<InteractiveDashboard dashboardId={dashboardId} />);
-    });
+  it("should have the correct tooltip position and z-index (metabase#51904, metabase#52732)", () => {
+    const testCases = [
+      // should use the user-supplied z-index
+      { input: 1337, expected: 1337 },
```

Comment:
We allow people to customize their `theme.components.popover.zIndex` in case some apps use a z-index that is higher than 200. This is the only aspect where the SDK test differs from the echarts tooltip helper assertions.